sunpy/timeseries/tests/test_timeseries_factory.py

```diff
@@ -273,20 +273,24 @@ def test_generic_construction_basic_different_meta_types(self):
         base = parse_time(datetime.datetime.today())
         times = base - TimeDelta(np.arange(24 * 60)*u.minute)
         intensity = np.sin(np.arange(0, 12 * np.pi, ((12 * np.pi) / (24*60))))
+        tr = sunpy.time.TimeRange('2012-06-01 00:00', '2012-06-02 00:00')
```

Comment:
what are the dates of this timerange for?

Should it not be between the start and end of `times` i.e. 
``` tr = sunpy.time.TimeRange(times[0], times[-1])```